src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -60,23 +89,19 @@
                                                   [:show-bottom-sheet {:content keypair-options}])}
        :description       :text
        :description-text  (i18n/label :t/keypairs-description)}]
-     [quo/keypair
-      {:customization-color customization-color
-       :profile-picture     profile-picture
-       :status-indicator    false
-       :type                :default-keypair
-       :stored              :on-device
-       :on-options-press    #(js/alert "Options pressed")
-       :action              :selector
-       :blur?               false
-       :details             {:full-name display-name
-                             :address   (utils/get-shortened-compressed-key compressed-key)}
-       :accounts            accounts
-       :container-style     {:margin-horizontal 20
-                             :margin-vertical   8}}]
+     [rn/flat-list
+      {:data                    keypairs
+       :render-fn               keypair
+       :render-data             {:profile-picture profile-picture
+                                 :compressed-key  compressed-key}
+       :content-container-style {:padding-bottom 60}}]
      [quo/bottom-actions
       {:actions          :one-action
        :button-one-label (i18n/label :t/confirm-account-origin)
        :button-one-props {:disabled?           true
                           :customization-color customization-color}
        :container-style  style/bottom-action-container}]]))
+
+(defn view
+  []
+  [:f> view-internal])
```

Comment:
I think we don't need this anymore.
cc @flexsurfer - perhaps we need to broadcast this info out to Status mobile devs now 👍